.ado/windows-vs-pr.yml

```diff
@@ -97,8 +97,8 @@ jobs:
           configuration: $(BuildConfiguration)
           publishRunAttachments: true
           collectDumpOn: onAbortOnly
-          vsTestVersion: 15.0
-        condition: and(eq(variables.BuildPlatform, 'x64'), eq(variables.BuildConfiguration, 'Debug'))
+          vsTestVersion: latest
+        condition: and(ne(variables['BuildPlatform'], 'arm'), ne(variables['BuildPlatform'], 'arm64'))
```

Comment:
>[' [](start = 35, length = 2)

Just curious: why not to use the property syntax?

Reply:
Fixed.